rotkehlchen/db/dbhandler.py

```diff
@@ -1139,33 +1140,44 @@ def add_multiple_location_data(self, write_cursor: 'DBCursor', location_data: li
                     f' already existing timestamp {entry.time}.',
                 ) from e
 
+    def _get_addressbook_entries_from_account_data(
+            self,
+            account_data: list[BlockchainAccountData],
+    ) -> list[AddressbookEntry]:
+        """Get list of addressbook entries from the accounts in account_data"""
+        return [
+            AddressbookEntry(
+                address=entry.address,
+                name=entry.label if entry.label is not None else '',
```

Comment:
this is an anti-pattern of how AddressbookEntry is used. `name` is never null. just return entries that have label or am i missing something?

Reply:
Same as https://github.com/rotki/rotki/pull/8863#discussion_r1828358195